pyodata/v2/service.py

```diff
@@ -139,7 +140,7 @@ def makefile(self, *args, **kwargs):
         return ODataHttpResponse(
             dict(response.getheaders()),
             response.status,
-            response.read(len(data))  # the len here will give a 'big enough' value to read the whole content
+            response.read(sys.getsizeof(data))  # the getsizeof here will actually give a 'big enough' value to read the whole content
```

Comment:
Since this is a fix for a bug, could you please capture it with a test? 

Pls add new test to test_service_v2.py, ideally one that would reproduce the bug before your fix and pass after your fix. This is relevant to big binary file, but you don't need to commit such big file (pls don't, git repo will be smaller :) ), just generate it inside the test.